api/src/main/java/jakarta/enterprise/inject/spi/BeanManager.java

```diff
@@ -221,28 +221,6 @@ public interface BeanManager extends BeanContainer {
      */
     public int getInterceptorBindingHashCode(Annotation interceptorBinding);
 
-    /**
-     * Returns a {@link jakarta.el.ELResolver} that resolves beans by EL name.
-     *
-     * @deprecated use {@code ELAwareBeanManager}, this method will be removed in CDI 5.0
-     * @return the {@link jakarta.el.ELResolver}
-     */
-    @Deprecated(since = "4.1", forRemoval = true)
```

Comment:
Also remove imports related to `ElResolver` from this class.

Reply:
Good point, I forgot about that yesterday.

Reply:
Done.